composer.json

```
@@ -18,8 +18,8 @@
         "test": "vendor/bin/phpunit"
     },
     "require": {
-        "php": "^7.4|^8.0|^8.1|^8.2",
-        "brick/math": "^0.9.1|^0.10|^0.11",
+        "php": "^7.4|^8.0|^8.1|^8.2|^8.3",
+        "brick/math": "^0.9.1|^0.10|^0.11|^0.12",
```

```suggestion
        "brick/math": "^0.9.1|^0.10|^0.11|^0.12|^0.13",
```

https://github.com/brick/math/releases/tag/0.13.0